README.adoc

```diff
@@ -0,0 +1,69 @@
+[discrete]
+= aktualizr-lite
```

Comment:
You think doing this as Markdown would make sense? That's pretty much the defacto standard of most projects these days. and we aren't building docs from this like the aktualizr repo does?

Reply:
> You think doing this as Markdown would make sense?
I don't have a strong opinion. There are different opinions around, https://mister-gold.pro/posts/en/asciidoc-vs-markdown/, https://www.zverovich.net/2016/06/16/rst-vs-markdown.html. I have worked mostly with asciidoc and RST. I suppose if we are not going to create huge documentation then Markdown should suffice.